### PR TITLE
"Try Out" section updated for Alderaan

### PR DESCRIPTION
**Pull request by taleldayekh**
As outlined in issue #393

### CODE REVIEW

**Comment by taleldayekh** [comment]
I moved "Try Out the Raiden Demo App" to its own section instead of having it nested within the "Getting Started" section.

It felt a bit misplaced there because "Getting Started" is more about working with the code base. For instance right after the Demo App tutorial the following section would appear:
```To run the code in this repository, you must have Node.js 10+ on your computer and a web3-enabled browser (e.g. Firefox with Metamask extension), as well as some ETH on the account.```

File: README.md [hunk 1]
```diff
@@ -142,45 +142,42 @@ If you didn't use Raiden before, you can
...
+__Step 1: Acquire ETH__
+1. Visit the Goerli faucet at [htts://faucet.goerli.mudit.blog](https://faucet.goerli.mudit.blog)
+2. Follow the instructions on the faucet's website to acquire Goerli ETH
 
+__Step 2: Acquire TTT (TestToken)__
+1. Visit [this page on Etherscan](https://goerli.etherscan.io/address/0x3ed0daedc3217615bde34fedd023bc81ae49251b#writeContract) where you can write to the TTT contract
+2. Enter your MetaMask and choose the Goerli account with ETH
+3. Go back to Etherscan and click "Connect to Web3"
+4. Scroll down to the "mint" field and enter ```1000000000000000000000```
+5. Click the "Write" button and confirm the transaction in MetaMask
```
Comment (christianbrb) [inline README.md]: The user needs to check, if the transaction has been successful

File: README.md [hunk 2]
```diff
@@ -261,6 +219,45 @@ npm run serve
...
+## Try Out the Raiden Demo dApp
+
+These step-by-step instructions will guide you through the process for trying out the Raiden demo dApp. The dApp is hosted at [https://lightclient.raiden.network/](https://lightclient.raiden.network/) and we will be using the Goerli testnet and MetaMask wallet in this example.
+
+__Prerequisites__
+
+You need to have MetaMask installed for your browser. If you don't have MetaMask, [visit their website](https://metamask.io/) to download and install it.
+
+__Step 1: Acquire ETH__
+1. Go to the Goerli faucet at [htts://faucet.goerli.mudit.blog](https://faucet.goerli.mudit.blog)
```
Comment (christianbrb) [inline README.md]: You could also add this faucet: https://goerli-faucet.slock.it/

File: README.md (hunk 1 above)
Comment (taleldayekh) [reply README.md]: I added a line about successful transaction in the final bullet point. Also the gif I created yesterday shows this.

**Pull request merged**